api/bazel/repository_locations.bzl

```diff
@@ -88,4 +88,15 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2020-08-17",
         use_category = ["api"],
     ),
+    com_github_apache_skywalking_data_collect_protocol = dict(
+        project_name = "SkyWalking API",
+        project_desc = "SkyWalking's language independent model and GRPC Api Definitions",
+        project_url = "https://github.com/apache/skywalking-data-collect-protocol",
+        version = "8.1.0",
+        sha256 = "ebea8a6968722524d1bcc4426fb6a29907ddc2902aac7de1559012d3eee90cf9",
+        strip_prefix = "skywalking-data-collect-protocol-{version}",
+        urls = ["https://github.com/apache/skywalking-data-collect-protocol/archive/v{version}.tar.gz"],
+        release_date = "2020-07-29",
```

Comment:
UTC for this commit is `2020-07-31`

Reply:
https://dev.azure.com/cncf/envoy/_build/results?buildId=54896&view=logs&jobId=1c04237c-5f52-5959-5f83-5522b3556331&j=1c04237c-5f52-5959-5f83-5522b3556331&t=db3f3889-c922-566b-fa2e-4bc4948bc467 says 7/29. This seems too far out to be explained even by TZ difference. I wonder if there is some methodology issue in the release dates script. I think it's fine to have this merge as is, we need to fix the deps check tooling.

Reply:
If this is passing tests I would leave the `release_date` as it is. I have updated `release_dates.py` code that now correctly checks the publish time of the release vs. the last commit in the release.